flink-shaded-jackson-parent/flink-shaded-jackson-2/pom.xml

```diff
@@ -78,6 +78,9 @@ under the License.
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
+                <!-- Jackson uses multi release jars and in case 1.15.x there is version 19
```

Comment:
```suggestion
                <!-- Jackson uses multi release jars and in case 2.15.x there is version 19
```
@snuyanzin  that's a typo, isn't it?

Reply:
yes, you're right, thanks for highlighting I will submit a hotfix PR for that

Reply:
https://github.com/apache/flink-shaded/pull/132